python/function.py

```diff
@@ -567,6 +601,12 @@ def signed_range_value(self, ranges):
 
 	@classmethod
 	def unsigned_range_value(self, ranges):
+		"""
+		Create a PossibleValueSet object for a unsigned signed range of values.
+
+		:param list(ValueRange) ranges: List of ValueRanges
```

Comment:
See above about an example, same here, could use almost the same example but one showing the differences would be good.

Reply:
Right I can add an example. But we dont currently perform any processing of the `PossibleValueSet` at the API level and hence cant really showcase the difference of how `SignedRangeValue`s and `UnsignedRangeValue`s are processed.